packages/beacon-node/src/chain/bls/maybeBatch.ts

```diff
@@ -1,29 +1,18 @@
-import {CoordType, PublicKey} from "@chainsafe/bls/types";
 import bls from "@chainsafe/bls";
+import {WorkRequestSet} from "./multithread/types.js";
+import {deserializeSet} from "./multithread/utils.js";
 
 const MIN_SET_COUNT_TO_BATCH = 2;
 
-export type SignatureSetDeserialized = {
-  publicKey: PublicKey;
-  message: Uint8Array;
-  signature: Uint8Array;
-};
-
 /**
  * Verify signatures sets with batch verification or regular core verify depending on the set count.
  * Abstracted in a separate file to be consumed by the threaded pool and the main thread implementation.
  */
```

Comment:
drop a comment that this is for worker impl only and it's deprecated since v1.18, to be cleaned up later